config/v1/feature_gates.go

```diff
@@ -381,4 +381,14 @@ var (
 		ResponsiblePerson:   "djoshy",
 		OwningProduct:       ocpSpecific,
 	}
+
+	FeatureGateGCPCloudLBIPs = FeatureGateName("GCPCloudLBIPs")
```

Comment:
Prefer not to use acronyms where possible, what about
```suggestion
	FeatureGateGCPCloudLBIPs = FeatureGateName("GCPCloudLoadBalancerIPs")
```
Also, is this being used in conjunction with another feature gate at all? I know this has come out of the cluster hosted DNS effort, or is that already stable?

Reply:
https://github.com/openshift/api/blob/master/config/v1/feature_gates.go#L355-L363 is the other feature gate you are referring to. 
Yes, this is part of the `ClusterHostedDNS` effort. These LoadBalancer IPs are being introduced per platform. The MCO would refer to these while generating the CoreDNS CoreFile for the in-cluster DNS.